app/controllers/address_controller.rb

```diff
@@ -0,0 +1,35 @@
+class AddressController < ApplicationController
+  def create
+    user = User.find_by(verification_token: params[:verification_token])
+    address = user.addresses.build(permitted_address_params)
```

Comment:
let's create a method called `current_user` and use it everywhere to access the logged in user

Reply:
Already have this helper named logged_in_user, using this helper instead of finding user by verification_token